charts/jenkins/pod-templates/nuxeo-js-client.yaml.gotmpl

```diff
@@ -0,0 +1,44 @@
+- id: "nuxeo-js-client"
+  name: "nuxeo-js-client"
+  label: "jenkins-nuxeo-js-client"
+  imagePullSecrets:
+  - name: {{ .Values.podTemplates.imagePullSecretName }}
+  serviceAccount: {{ .Values.podTemplates.serviceAccount }}
+  nodeUsageMode:  {{ .Values.podTemplates.nodeUsageMode }}
+  containers:
+  {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
+  - args: ""
+    command: ""
+    image: "{{ .Values.docker.registry }}/nuxeo/builder-nodejs16:latest"
```

Comment:
```suggestion
    image: "{{ .Values.docker.registry }}/nuxeo/builder-nodejs18:latest"
```

Reply:
This is currently not possible, see https://github.com/nuxeo/jx-platform-builders/pull/71.

We will do the upgrade after switch to rocky linux distribution, probably around the end of life of nodejs 14.